### PR TITLE
chore: replacing hard coded string

### PR DESCRIPTION
**Pull request by sondermanish**
#### Type of change
- Chore (housekeeping or task changes that don't impact user perception)

#### How Has This Been Tested?
- [x] Manual

#### Dev activity
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] PR is being merged under a feature flag

### CODE REVIEW

File: app/server/appsmith-server/src/main/java/com/appsmith/server/constants/ce/FieldNameCE.java [hunk 1]
```diff
@@ -177,4 +177,12 @@ public class FieldNameCE {
     public static final String UNUSED_ENVIRONMENT_ID = "unused_env";
 
     public static final String SUFFIX_USER_MANAGEMENT_ROLE = " User Management";
+
+    //short names for analytics
```
Comment (nidhi-nair) [inline app/server/appsmith-server/src/main/java/com/appsmith/server/constants/ce/FieldNameCE.java:181]: Let's make a separate constants class for Analytics? Shouldn't crowd a single class with all constants.

**Review by subrata71 — APPROVED** [review]
LGTM

**Pull request merged**